### DISCUSSION TITLE
Articles and Resources to Learn TypeGraphQL

### DISCUSSION THREAD

**Discussion by codingknite** [EddieHubCommunity/support#659] [General]
Here are a few resources that I found that can help anyone who wishes to learn or read about TypeGraphQL

[TypeGraphQL Documentation](https://typegraphql.com/docs/getting-started.html)
[Article On TypeGraphQL](https://maximefleurant.com/compute/typegraphql)
[Ben Awad's Series on TypeGraphQL](https://www.youtube.com/watch?v=8yZImm2A1KE&list=PLN3n1USn4xlma1bBu3Tloe4NyYn9Ko8Gs&ab_channel=BenAwad)

Feel free to add any that you might know of

**Comment by eddiejaoude** [comment]
Thanks for sharing 👍
